plugins/modules/system/filesystem.py

```diff
@@ -369,34 +391,35 @@ def main():
     # There is no "single command" to manipulate filesystems, so we map them all out and their options
     module = AnsibleModule(
         argument_spec=dict(
-            fstype=dict(required=True, aliases=['type'],
-                        choices=list(fstypes)),
+            state=dict(default='present', choices=['present', 'absent']),
```

Comment:
```suggestion
            state=dict(type='str', default='present', choices=['present', 'absent']),
```

Reply:
why especially this one would have an explicit type, since `str` is the default and other string params (`fstype`, `dev` and `opts`) use this implicit type ?

Reply:
Basically because IMO explicit is better than implicit :) (I would also change the other ones, but doing too much in one PR is never a good idea.)

If you want you can leave the type away here.

Reply:
I prefer to be consistent with this kind of formalisms. Explicit everywhere, or as implicit (light) as possible, it could be equal to me, as long as the same rules are applied in a same file. So I prefer to decline your suggestion.